.github/workflows/nuget_slack_notifications.yml

```diff
@@ -33,7 +33,15 @@ jobs:
       - name: Harden Runner
         uses: step-security/harden-runner@03bee3930647ebbf994244c21ddbc0d4933aab4f # v2.3.0
         with:
-          egress-policy: audit # TODO: change to 'egress-policy: block' after couple of runs
+          disable-sudo: true
+          egress-policy: block
+          allowed-endpoints: >
+            api.nuget.org:443
+            collector-1.staging-collectors.newrelic.com:443
```

Comment:
This could be a problem if the initial call to the collector (`staging-collector.newrelic.com`) returns something other than `collector-1....` for the actual collector instance the agent should talk to.  Can this action accept wildcards in allowed-endpoints?

FYI, the only reason this job needs to talk to the collector is because I got the bright idea back in November to instrument the Dotty process with the agent to detect errors.  If it becomes a lot of toil to maintain we can just undo that change.

Reply:
Good question. I'll see what I can do.

Reply:
This sounds like something we'll need to solve for the integration tests anyways.

Reply:
Nope, doesn't look like they support wildcards. I'm starting to wonder if using this action is really going to be worth the effort - I definitely don't want to constantly have to come back and tweak the configuration...

Reply:
Hi @tippmar-nr, I am the one who had emailed you :smile:. For most jobs, the outbound connections don't change often. If you have a job where the outbound calls might be different in each run, you can still run it in `audit` mode, so if there is something odd and you need to investigate, you at least have the insights data (outbound calls/ file overwrites) for analysis. 

We have an [issue related to supporting wildcards](https://github.com/step-security/harden-runner/issues/236) and I can let you know once that is done. We also support storing the policy in a [policy store](https://docs.stepsecurity.io/harden-runner/how-tos/block-egress-traffic#2-add-the-policy-using-the-policy-store) so you can change it without the need to change the workflow file. 

Let me know if you have any other feedback, and I can create an issue to track it. Thanks!

Reply:
Thanks @varunsh-coder - I'm updating to leave the action in `audit` mode. Will make this change in a few other places where we had otherwise disabled the harden-runner action.